packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/clusters_keeper/ec2_instances.py

```diff
@@ -0,0 +1,16 @@
+from models_library.api_schemas_clusters_keeper import CLUSTERS_KEEPER_RPC_NAMESPACE
+from models_library.api_schemas_clusters_keeper.ec2_instances import EC2InstanceType
+from models_library.rabbitmq_basic_types import RPCMethodName
+
+from ....rabbitmq import RabbitMQRPCClient
+
+
+async def get_instance_type_details(
+    client: RabbitMQRPCClient, *, instance_type_names: set[str]
+) -> list[EC2InstanceType]:
+    instance_types: list[EC2InstanceType] = await client.request(
```

Comment:
Q: The issue of the exceptions is still fully unsolved. AFAIK this function can only raise one type of exception, right? Perhpaps we can document it and even expose as `rpc_interfaces.errors`?

Reply:
I know of someone that always says: *one step at a time*
good point, will add it